lib/portlayer/portlayer.go

```diff
@@ -86,17 +89,21 @@ func Init(ctx context.Context, sess *session.Session) error {
 		return err
 	}
 
+	if err = logging.Init(ctx); err != nil {
+		return err
+	}
+
 	// Unbind containerVM serial ports configured with the old VCH IP.
 	// Useful when the appliance restarts and the VCH has a different IP.
```

Comment:
We should update this comment as well, now that we've changed `unbindSerialPorts` to `TakeCareOfSerialPorts`.